docs/getting_started/run_schedule.md

```diff
@@ -294,7 +294,7 @@ remote.client.update_launch_plan(launchplan_id, "ACTIVE")
 
 :::{code-block} bash
 flytectl update launchplan -p flyteexamples -d development \
-    process_data_lp --version <VERSION> --activate
+ process_data_lp --version <VERSION> --activate
```

Comment:
consider four spaces instead of one for more prominent indentation

Reply:
I enabled the black formatter, and it's automatically removing the spaces.